mslib/mswms/generics.py

```diff
@@ -303,6 +303,31 @@ def get_log_levels(cmin, cmax, levels=None):
     return clev
 
 
+
+CBAR_LABEL_FORMATS = {
+    "log": "%.3g",
+    "log_ice_cloud": "%.0E",
+}
+
+
+def get_cbar_label_format(style, maxvalue):
+    if style in CBAR_LABEL_FORMATS:
```

Comment:
we have humanfriendly in our package. Does this what you want?

https://humanfriendly.readthedocs.io/en/latest/readme.html#getting-started

Reply:
Maybe. I'll have a look at that. The major part of this PR is, however, to put it under user control, not the actual behaviour, which human-frienldy likely improves.